packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -4756,161 +5146,187 @@ curl -v -X POST \
 }
 ```
 
-#### Verify Call Factor
+#### Verify WebAuthn Factor
 
 
 <ApiOperation method="post" url="/api/v1/authn/factors/${factorId}/verify" />
 
-##### Request Parameters for Verify Call Factor
-
-
-| Parameter      | Description                                         | Param Type | DataType | Required |
-| -------------- | --------------------------------------------------- | ---------- | -------- | -------- |
-| factorId       | `id` of factor                                      | URL        | String   | TRUE     |
-| stateToken     | [state token](#state-token) for current transaction | Body       | String   | TRUE     |
-| passCode       | OTP sent to device                                  | Body       | String   | FALSE    |
-| rememberDevice | user's decision to remember device                  | URL        | Boolean  | FALSE    |
-
-> If you omit `passCode` in the request a new OTP will be sent to the device, otherwise the request will attempt to verify the `passCode`
-
-##### Response Parameters for Verify Call Factor
-
-
-[Authentication Transaction Object](#authentication-transaction-model) with the current [state](#transaction-state) for the authentication transaction.
-
-If the `passCode` is invalid you will receive a `403 Forbidden` status code with the following error:
+Verifies a user with a WebAuthn factor. The verification process starts with getting the WebAuthn credential request options, which are used to help select an appropriate authenticator using the WebAuthn API.
+This authenticator then generates an assertion which may be used to verify the user.
 
-```json
-{
-  "errorCode": "E0000068",
-  "errorSummary": "Invalid Passcode/Answer",
-  "errorLink": "E0000068",
-  "errorId": "oaei_IfXcpnTHit_YEKGInpFw",
-  "errorCauses": [
-    {
-      "errorSummary": "Your answer doesn't match our records. Please try again."
-    }
-  ]
-}
-```
+##### Start Verification to Get Challenge Nonce
 
-##### Send Voice Call Challenge (OTP)
+Verification of the WebAuthn factor starts with getting the WebAuthn credential request details (including the challenge nonce) then using the client-side
+JavaScript API to get the signed assertion from the WebAuthn authenticator.
 
-Omit `passCode` in the request to sent an OTP to the device.
+For more information about these credential request options, see the [WebAuthn spec for PublicKeyCredentialRequestOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions)
```

Comment:
add a period 

For more information about these credential request options, see the [WebAuthn spec for PublicKeyCredentialRequestOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions).